File: FootballApp/Models/Season.cs
namespace FootballApp.Models
{
    public class Season
    {
        public int Id { get; set; }
        public string Year { get; set; }
        public int SeasonStatusId { get; set; }
        public SeasonStatus SeasonStatus { get; set; }
        public bool HasTeams { get; set; }
        public int LeagueId { get; set; }
        public League League { get; set; }
    }
}

File: FootballApp-SPA/src/app/_models/Season.ts
import { League } from './league';
import { SeasonStatus } from './SeasonStatus';

export interface Season {
    id: number;
    year: string;
    seasonStatusId: number;
    seasonStatus: SeasonStatus;
    hasTeams: boolean;
    leagueId: number;
    league: League;
}
